src/components/Log/Log.js

```diff
@@ -43,9 +43,7 @@ class Log extends Component {
           <SkeletonText paragraph width="30%" />
         ) : (
           <>
-            <Ansi className="log-content" linkify>
-              {logs.join('\n')}
-            </Ansi>
+            <Ansi className="log-content">{logs.toString()}</Ansi>
```

Comment:
`logs` should be a string already

may need to update defaultProps + story

Reply:
I get a weird error from the Ansi component when I don't add toString, `Error: Uncaught [TypeError: txt.split is not a function]`